tools/restatectl/src/connection.rs

```diff
@@ -33,17 +33,22 @@ use crate::util::grpc_channel;
 
 #[derive(Clone, Parser, Collect, Debug)]
 pub struct ConnectionInfo {
-    /// Specify the nodes addresses separated by ','. This option can be specified multiple
-    /// times.
+    /// Specify server address to connect to.
+    ///
+    /// It needs access to the node-to-node address (aka. node advertised address)
+    /// Can also accept a comma-separated list or by repeating `--address=<host>`.
```

Comment:
If `--server` is the visible alias, should this rather be?
```suggestion
    /// Can also accept a comma-separated list or by repeating `--server=<host>`.
```

Reply:
it's still `--address`. `--server` is just an alias (not _the_ visible alias) as you can see in the screenshot.
<img width="818" alt="image" src="https://github.com/user-attachments/assets/e1881c78-7bc1-4f1d-a139-35d406a23493" />